server/controllers/review.js

```diff
@@ -42,6 +45,26 @@ exports.createReview = asyncHandler(async (req, res, next) => {
   }
 });
 
+exports.getAvgUserReview = asyncHandler(async (req, res, next) => {
+  Review.aggregate(
+    [
+      { $group: { _id: "$userId", avgRating: { $avg: "$rating" } } }
+    ]
```

Comment:
This is getting the average rating of all users, this is not the best approach. I don't think having a specific route for this is ideal, would be better to have a way to populate this value in the profile when fetching the profile